tests/test_util.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+from numpy.testing import assert_allclose
+import pytest
+import sfs
+
+test_data = [
+    ((1, 1, 1), (np.pi / 4, np.arccos(1/np.sqrt(3)), np.sqrt(3))),
+    ((-1,1,1), (np.arctan2(1,-1), np.arccos(1/np.sqrt(3)), np.sqrt(3))),
+    ((1,-1,1), (-np.pi / 4, np.arccos(1/np.sqrt(3)), np.sqrt(3))),
+    ((-1,-1,1), (np.arctan2(-1,-1), np.arccos(1/np.sqrt(3)), np.sqrt(3))),
+    ((1,1,-1), (np.pi / 4, np.arccos(-1/np.sqrt(3)), np.sqrt(3))),
+    ((-1,1,-1), (np.arctan2(1,-1), np.arccos(-1/np.sqrt(3)), np.sqrt(3))),
+    ((1,-1,-1), (-np.pi / 4, np.arccos(-1/np.sqrt(3)), np.sqrt(3))),
+    ((-1,-1,-1), (np.arctan2(-1,-1), np.arccos(-1/np.sqrt(3)), np.sqrt(3)))
```

Comment:
When having a list with one entry per line, it's considered good style to add a final comma to the last line, even though this is not strictly necessary. This makes it easier to add new lines later. It also makes it easier to re-arrange the lines if that's desired at some point.

Reply:
Thank you for the feedback. I will explore pycodestyle to correct the coding style.

about assert_allclose(), I had a precision higher than the default so I decided to exploit it.